recipes/anvio-minimal/meta.yaml

```diff
@@ -12,43 +12,38 @@ source:
 
 build:
   number: 0
+  skip: True  # [not py36]
   script: python -m pip install --no-deps --ignore-installed .
 
 requirements:
-  build:
-    - {{ compiler('c') }}
   host:
-    - python >=3,<3.7
-    - pip
+    - python
```

Comment:
Does this still not work with python 3.7?

Reply:
Unfortunately not, I tried with the newest versions of our dependencies, some does not support Python 3.7. Expectedly some older versions also do not work on 3.7. So we had to manually change versions to find a working combination. We pinned the first combination we got it to work.

Reply:
Do you recall which of you dependencies do not support 3.7. I guess the package is just missing for 3.7 and we can fix this package instead.

Reply:
I'll tweak the recipe and have it test for the python 3.7 issue.